.gitignore

```diff
@@ -50,3 +50,4 @@ libsnark/zk_proof_systems/zksnark/ram_zksnark/tests/test_ram_zksnark
 
 build
 *~
```

Comment:
Missing newline at the end of the file: https://thoughtbot.com/blog/no-newline-at-end-of-file

Reply:
Done. Also added (setq require-final-newline t) in my dotemacs to prevent similar such omissions in the future.